questions/qa-date-format.en.html

```diff
@@ -3,15 +3,15 @@
 <head>
 <meta charset="utf-8" />
 <title>Date formats</title>
-<meta name="description" content="How do I prepare my web pages to display varying international date formats?" />
+<meta name="description" content="How do I manage date formats on a web page?" />
```

Comment:
This review focuses mainly on the first section of the document.  There are lots of comments: don't feel overwhelmed.  I'm basically trying to understand what is the need of the content author that we are addressing, and how best to meet that/those need(s).  The initial scope of the article was fairly limited and specific (albeit perhaps in need of updating).  We've widened that scope significantly here and we need to, i think, in our minds have more clear vision of what we are trying to achieve and more rigorous control over what text we use to achieve the goal.  Part of that involves also deciding to what exent we describe the how-to steps in this article and give example, compared with pointing off to other sources for those things.